.gitlab-ci.yml

```diff
@@ -102,6 +102,7 @@ before_script:
   interruptible: true
   dependencies: []
   script:
+    - ulimit -s 16384 # flambda can be pretty stack hungry, specially with -O3
```

Comment:
I guess that's OK

Reply:
See also https://github.com/ocaml/ocaml/issues/7842#issuecomment-596863244 , I plan to raise the severity of this.

Reply:
Will add that link to the comment in fact.